samcli/local/common/runtime_template.py

```diff
@@ -133,6 +135,7 @@ def get_local_lambda_images_location(mapping, runtime):
 ]
 
 LAMBDA_IMAGES_RUNTIMES = [
+    "amazon/nodejs14.x-base",
```

Comment:
This corresponds to an app template, we should make sure that we ship that part first before releasing SAM CLI. Otherwise this will fail for customers until that is out.